include/circt/Dialect/SV/SVInOutOps.td

```diff
@@ -137,3 +137,20 @@ def ArrayIndexInOutOp
   let assemblyFormat =
     "$input`[`$index`]` attr-dict `:` type($input) `,` type($index)";
 }
+
+def PartSelectOp : SVOp<"part_select", [NoSideEffect]> {
+  let summary = "Address several contiguous bits of an inout type (e.g. a wire"
+                " or inout port). This is an indexed part-select operator."
+                "The base is an integer expression and the width is an "
+                " integer constant. The bits start from base and the number "
+                "of bits selected is equal to width. See SV Spec 11.5.1.";
+  let arguments = (ins InOutType:$input, HWIntegerType:$base, I32Attr:$width);
```

Comment:
This seems to be the +: case, can you add an optional keyword to the op for the -: case?

Reply:
Added the decrement case, `-` is not a valid token in optional assembly format, so added the `decrement` keyword instead.